test/expected_outputs/green-proof02/virtual_services-ingress.rendered

```diff
@@ -7,8 +7,8 @@ spec:
   gateways:
     - default-gateway
   hosts:
-    - apps.green.wealthwizards.io
     - "*.apps.green.wealthwizards.io"
+    - "apps.green.wealthwizards.io"
```

Comment:
Because I'm determine this list automatically the order changes and we treat each entry the same, hence the quotes being added here. This should change the behaviour.